views/shared/flash.html

```diff
@@ -0,0 +1,14 @@
+{{ if .flash }}
+    <div class="flash">
+        {{ if .flash.success }}
+            <div class="flash__success">
+                {{ .flash.success }}
+            </div>
+        {{ end }}
+        {{ if .flash.error }}
+            <div class="flash__error">
+                {{ .flash.error }}
+            </div>
+        {{ end }}
```

Comment:
@junan Will this works?
```suggestion
        {{ range $type, $message := .flash }}
            <div class="flash__{{$type}}">
                {{ $message }}
            </div>
        {{ end }}
```

Reply:
It works, thanks